datatable_tutorial/datatable_tutorial/datatable_tutorial.py

```diff
@@ -175,95 +231,131 @@ def toggle_pause(self):
+                                    on_delete=DataTableState.get_deleted_item,
+                                    # on_row_appended=DataTableState.append_row,
+                                    on_column_resize=DataTableState.column_resize,
+                                    theme=darkTheme,
+                                    width="80vw",
+                                    height="80vh",
+                                ),
                             ),
                         ),
-                        rx.data_editor(
-                            columns=DataTableLiveState.columns,
-                            data=DataTableLiveState.table_data,
-                            draw_focus_ring=True,
-                            row_height=50,
-                            smooth_scroll_x=True,
-                            smooth_scroll_y=True,
-                            column_select="single",
-                            # style
-                            theme=darkTheme,
+                        rx.tab_panel(
+                            rx.vstack(
+                                rx.stack(
+                                    rx.cond(
+                                        ~DataTableLiveState.running,
+                                        rx.button(
+                                            "Start",
+                                            on_click=DataTableLiveState.toggle_pause,
+                                            color_scheme="green",
+                                        ),
+                                        rx.button(
+                                            "Pause",
+                                            on_click=DataTableLiveState.toggle_pause,
+                                            color_scheme="red",
+                                        ),
+                                    ),
+                                ),
+                                rx.data_editor(
+                                    columns=DataTableLiveState.columns,
+                                    data=DataTableLiveState.table_data,
+                                    draw_focus_ring=True,
+                                    row_height=50,
+                                    smooth_scroll_x=True,
+                                    smooth_scroll_y=True,
+                                    column_select="single",
+                                    # style
+                                    theme=darkTheme,
+                                ),
+                                overflow_x="auto",
+                                width="100%",
                             ),
-                        overflow_x="auto",
-                        width="100%",
-                        height="40vh",
+                        ),
                     ),
+                    spacing="1.5em",
+                    font_size="2em",
+                    padding_top="10%",
```

Comment:
```suggestion
                    padding_top="10vh",
                    width="90vw",
```

tab bar looks a bit better with viewport units, doesn't jump around when changing tabs (as much)